src/Avalonia.Diagnostics/DevToolsExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Avalonia.Controls;
+using Avalonia.Diagnostics;
+using Avalonia.Input;
+
+namespace Avalonia
+{
+    public static class DevToolsExtensions
```

Comment:
Is it the time we start documenting public APIs ? :)

Reply:
It is, yes! :)